netci.groovy

```diff
@@ -912,7 +920,7 @@ combinedScenarios.each { scenario ->
         // Architectures.  x64 only at this point
         ['x64'].each { architecture ->
             // Put the OS's supported for coreclr cross testing here
-            Constants.crossList.each { os ->
```

Comment:
@mmitche I was considering that, but I don't want to, for example, add daily CentOS JIT stress test runs, which adding CentOS to crossList would do (without further refactoring).